src/plugins/share/public/components/tabs/export/export_content.tsx

```diff
@@ -51,8 +51,12 @@ const ExportContentUi = ({
     return aggregateReportTypes
       .filter(({ reportType }) => reportType)
       .map(({ reportType, label }) => {
-        return { id: reportType, label, 'data-test-subj': `${reportType}-radioOption` };
-      }) as EuiRadioGroupOption[];
+        return {
+          id: reportType!,
```

Comment:
nit: will be perfect if we add a typeguard here in the filter so we don't have to add the non-null assertion